RX.h

```diff
@@ -9,6 +9,11 @@ uint32_t lastPacketTimeUs = 0;
 uint32_t lastRSSITimeUs = 0;
 uint32_t linkLossTimeMs;
 
+uint32_t hopInterval = 0;
+uint32_t hopTimeout = 0;
+uint32_t hopTimeoutSlow = 0;
+uint32_t RSSI_timeout = 0;
```

Comment:
ok, I'm cool with that...
yeah, I don't personally like doing the underscore thing, but I was trying to emulate what I had seen elsewhere in kha's code.

I'll try to get it fixed&pushed later today.